client/src/MainMenu.tsx

```diff
@@ -1,22 +1,45 @@
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useLocation } from 'react-router-dom';
 
-export function MainMenu(): JSX.Element {
+const MainMenu: React.FC = () => {
+  const location = useLocation();
+  const { pathname } = location;
+  const [activeTab, setActiveTab] = React.useState<string>(
+    pathname.split('/')[1],
+  );
   const tabs = [
     { linkTo: '/users', label: 'Users' },
     { linkTo: '/applications', label: 'Applications' },
     { linkTo: '/companies', label: 'Companies' },
+    { linkTo: '/companies_es', label: 'Companies', new: 'ES' },
     { linkTo: '/payments', label: 'Payments' },
     { linkTo: '/customers', label: 'Customers' },
     { linkTo: '/admins', label: 'Admins' },
     { linkTo: '/clients', label: 'Clients' },
-  ].map((tab, index) => {
-    return (
-      <li role="presentation" key={index}>
-        <Link to={tab.linkTo}>{tab.label}</Link>
-      </li>
-    );
-  });
+  ];
 
-  return <ul className="nav nav-tabs">{tabs}</ul>;
-}
+  React.useEffect(() => {
+    setActiveTab(pathname.split('/')[1]);
```

Comment:
```suggestion
    pathname.split('/')[1]
```

Would be nice to wrap this with a meaningful function, i.e getTabName, or sth